packages/react-aria-components/docs/Tabs.mdx

```diff
@@ -691,39 +702,6 @@ import {Link} from 'react-aria-components';
 </Router>
 ```
 
-<details>
```

Comment:
I do think the links look a bit large...maybe we should change it like you suggested